joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/SetterTests.scala

```diff
@@ -1,29 +1,33 @@
 package io.joern.rubysrc2cpg.querying
 
 import io.joern.rubysrc2cpg.testfixtures.RubyCode2CpgFixture
+import io.shiftleft.codepropertygraph.generated.Operators
+import io.shiftleft.codepropertygraph.generated.nodes.{Call, FieldIdentifier, Identifier}
 import io.shiftleft.semanticcpg.language.*
 
 class SetterTests extends RubyCode2CpgFixture {
 
-  "`x.y=1` is represented by a `x.y=` CALL with argument `1`" in {
+  "`x.y=1` is approximated by a `x.y =` assigment with argument `1`" in {
```

Comment:
Why is there only a test for the setter? This also needs to apply to the getter side.

Reply:
Would this still be a call or a field getter?

Reply:
I suppose let's keep it a call, otherwise we have to deal with parenthesis-less call ambiguity